src/logic/map_objects/map_object_program.cc

```diff
@@ -367,29 +368,72 @@ MapObjectProgram::AnimationParameters MapObjectProgram::parse_act_animate(
 	return result;
 }
 
+/* RST
+
+.. _map_object_programs_playsound:
+
+playsound
+^^^^^^^^^^
+.. function:: playsound=\<sound_dir/sound_name\> priority:<\percent\>
+
+   :arg string sound_dir/sound_name: The directory (folder) that the sound files are in,
+      relative to the data directory, followed by the name of the particular sound to play.
+      There can be multiple sound files to select from at random, e.g.
+      for `sound/farm/scythe`, we can have `sound/farm/scythe_00.ogg`, `sound/farm/scythe_01.ogg`
+      ...
+
+   :arg percent priority: The priority to give this sound,
+      in :ref:`map_object_programs_datatypes_percent`. Maximum priority is ``200%``.
+
+   Trigger a sound effect. Whether the sound effect is actually played is determined by the
+   sound handler.
+   For priorities ``<= 100%``, this is the direct chance to be played, but only if this sound
+   effect is not already playing.
+   For priorities ``> 100%``, ``100`` is subtracted from the chance (so that ``101%-200%`` given
+   corresponds to an actual chance of ``1%-100%``), and the same sound effect can be played at the
+   same time for multiple map objects.
```

Comment:
This is not intuitive. IMHO it would be better to use the range `1%`-`100%` and have an additional boolean flag `allow_multiple:<true|false>`

Reply:
Good idea, will do that :)

Reply:
How about renaming the parameter according to single/multiple? So, we could have:

Design 1:

```
playsound=sound/farm/horse single:50%
playsound=sound/farm/horse multiple:50%
```

Or with 3 parameters, we would have:

Design 2:


```
playsound=sound/farm/horse chance:50% allow_multiple:false
playsound=sound/farm/horse chance:50% allow_multiple:true
```

Or make the 3rd parameter optional:

Design 3:

```
playsound=sound/farm/horse chance:50%
playsound=sound/farm/horse chance:50% allow_multiple
```
I like Design 3, but you are welcome to overrule me on this one.

@hessenfarmer @MarkMcWire

Reply:
Design 3 is up